plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiMetadata.java

```diff
@@ -72,6 +73,8 @@
 public class HudiMetadata
         implements ConnectorMetadata
 {
+    public static final Logger log = Logger.get(HudiMetadata.class);
```

Comment:
How is the removal of the warning related to removing the dependency towards `hudi-hive-sync` ?